scripts/background.js

```diff
@@ -14,7 +14,7 @@ var VERSION = manifest.version
 var globalStatusCode = ''
 let toolbarIconState = {}
 let tabIdPromise
-var WB_API_URL = 'https://archive.org/wayback/available'
+var WB_API_URL = hostURL+'wayback/available'
```

Comment:
Already merged, but found one possible bug here, as this may be defined before hostURL is?

Reply:
@cgorringe Actually utils.js is loaded before background.js . You can check in manifest.json file lineNo-37